exporter/awsemfexporter/emf_exporter.go

```diff
@@ -32,7 +32,7 @@ const (
 
 	// AppSignals EMF config
 	appSignalsMetricNamespace    = "AppSignals"
```

Comment:
should we also update Metric namespace here?

Reply:
looks like we will have to change it, it is used to determine whether we should inject AppSignals flag in useragent

Reply:
Good catch, updated.